minifi-nar-bundles/minifi-provenance-reporting-bundle/pom.xml

```diff
@@ -25,10 +25,17 @@
     <packaging>pom</packaging>
 
     <modules>
-        <module>minifi-provenance-reporting-task</module>
         <module>minifi-provenance-reporting-nar</module>
     </modules>
 
+    <dependencies>
+        <dependency>
+            <groupId>org.apache.nifi</groupId>
+            <artifactId>nifi-site-to-site-reporting-task</artifactId>
+            <version>1.0.0</version>
+        </dependency>
+    </dependencies>
+
     <dependencyManagement>
```

Comment:
I believe this dependency management section can be removed (or at least remove minifi-provenance-reporting-task since it no longer exists).